tests/jest/config/webpack/loaders/__snapshots__/devSitePackage.test.jsx.snap

```diff
@@ -0,0 +1,57 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`devSitePackage loader test when both repository url and repository directory are specified 1`] = `
+"import React from 'react';
+import Badges from 'terra-dev-site/lib/loader-components/_Badges';
+
+export const Badge = ({ url }) => (
+      <Badges
+        src=\\"https://github.com/cerner/terra-dev-site/tree/master/src/loader-components\\"
+        name=\\"terra-dev-site\\"
+        version=\\"6.2.0\\"
+        url={url}
+      />
+    );"
+`;
+
+exports[`devSitePackage loader test when only repository directory is specified 1`] = `
+"import React from 'react';
+import Badges from 'terra-dev-site/lib/loader-components/_Badges';
+
+export const Badge = ({ url }) => (
+      <Badges
+        src=\\"\\"
```

Comment:
is this correct?

Reply:
Yes, In the scenario when the repository url is not specified, the `src` should be passed as an empty string.

Reply:
I would expect the src key to not be pass at all if the url is not specified

Reply:
Changed here : https://github.com/cerner/terra-dev-site/pull/243/commits/3acf9f958a72e6ad031400f2c566813e15a3c78d